README.md

```diff
@@ -215,6 +229,38 @@ client.checkout.removeDiscount(checkoutId).then(checkout => {
 });
 ```
 
+## Expanding the SDK
+
+Not all fields that are available on the [Storefront API](https://help.shopify.com/en/api/custom-storefronts/storefront-api/reference) are exposed through the SDK. If you use the unoptimized version of the SDK, you can easily build your own queries. In order to do this, use the UMD Unoptimized build.
+
+### Initializing the Client
```

Comment:
This should specify which client. By default, we want to recommend the optimized client, but for special cases, use the thicker one.

Reply:
(I know it's under the expanding the sdk section. But for the sake of people skimming or looking at just the headings it'd be useful)